Vote/VoteAccessor.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/*
+ * This file is part of the FOSCommentBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace FOS\CommentBundle\Vote;
+
+use FOS\CommentBundle\Model\VotableCommentInterface;
+use FOS\CommentBundle\Model\VoteManagerInterface;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+class VoteAccessor implements VoteAccessorInterface
```

Comment:
Current class is necessary to avoid unnecessary queries when constructing votes. We select all the votes of the current user in the comments thread and cached with for following calls.
I'm not sure about the class name 😄